rules/library.bzl

```diff
@@ -945,6 +945,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
                 module_name = module_name,
                 tags = _MANUAL,
             )
+            objc_hdrs.append(module_map)
```

Comment:
You don't want the extended module map here intentionally, right?

Reply:
I'll try to circle back on this soon, it looks like we're having cache invalidation issues on a lot of these 🤔